### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by holly-cummins** [quarkiverse/antora-ui-quarkiverse#68]
See https://github.com/quarkusio/quarkusio.github.io/issues/1809#issuecomment-1979565495 and https://github.com/quarkusio/quarkusio.github.io/pull/1908.

**Comment by cescoffier** [comment]
Yes, please provide dark/light/system selector with a clear CSS class on the body to adapt the style
